pkg/controller/migmigration/stage.go

```diff
@@ -340,13 +382,17 @@ func (t *Task) ensureStagePodsFromRunning() error {
 	if err != nil {
 		return liberr.Wrap(err)
 	}
+	stagePodImage, err := t.getSrcStagePodImage()
+	if err != nil {
+		return liberr.Wrap(err)
+	}
 	for _, ns := range t.sourceNamespaces() {
 		podList := corev1.PodList{}
 		err := client.List(context.TODO(), k8sclient.InNamespace(ns), &podList)
 		if err != nil {
 			return liberr.Wrap(err)
 		}
-		stagePods.merge(BuildStagePods(t.stagePodLabels(), t.getPVCs(), &podList.Items, resourceLimitMapping)...)
+		stagePods.merge(BuildStagePods(t.stagePodLabels(), t.getPVCs(), &podList.Items, resourceLimitMapping, stagePodImage)...)
```

Comment:
@sseago please verify that I'm using the correct stage pod image here (srcCluster)